docs/cargo.md

```diff
@@ -0,0 +1,66 @@
+# Cargo
+
+<https://doc.rust-lang.org/cargo/>
+
+## Prerequisites
+
+To use Cachi2 with Cargo locally, ensure you have Cargo binary installed on your
+system. Then, ensure that the **Cargo.toml** and **Cargo.lock** are in your
+project directory.
+
+## Usage
+
+Run the following commands in your terminal to prefetch your project's
+dependencies specified in the **Cargo.lock**. It must be synchronized with the **Cargo.toml**
+file. Otherwise, the command will fail.
+
+```bash
+cd path-to-your-rust-project
+cachi2 fetch-deps cargo
+```
+
+The default output directory is `cachi2-output`. You can change it by passing
+the `--output-dir` option for the `fetch-deps` command. See the help message
+for more information.
+
+After prefetching the dependencies, you can use the prefetched dependencies
+to build your project. The following command will update the `.cargo/config.toml`
+to use them. (If the file does not exist, it will be created).
```

Comment:
> The following command will update the ...

I think ^this wording is too soft, meaning that it doesn't sound like something mandatory for the user to do. They won't learn about this until a few paragraphs later mentioned below in the 

> \#\# Hermetic build

section. I would personally go for something stronger, e.g. _Make sure you run the command bellow which ..._

Also, I think we can be a tiny bit more explicit about the modified TOML config file being modified in their own repo and say: _... will update the `.cargo/config.toml in your repository to use them._